projects/ad7134_iio/app/app_config.c

```diff
@@ -172,7 +170,7 @@ static struct eeprom_24xx32a_init_param eeprom_extra_init_params = {
 
 /* EEPROM init parameters */
 static struct no_os_eeprom_init_param eeprom_init_params = {
-	.device_id = 0,
+	.device_id = I2C_DEVICE_ID,
```

Comment:
The macro would also need to be added in app_config_mbed.h I think?

Reply:
It is already present in the mbed config-https://github.com/analogdevicesinc/precision-converters-firmware/blob/release-ad777x-h563/projects/ad7134_iio/app/app_config_mbed.h#L85

Reply:
My bad, missed that somehow